devicetypes/smartthings/motion-detector.src/motion-detector.groovy

```diff
@@ -17,6 +17,8 @@ metadata {
 		capability "Sensor"
 
 		fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0001,0003,0009,0500"
+		//raw description 23 0104 0402 00 06 0000 0001 0003 000F 0020 0500 02 000A 0019
+		fingerprint endpoint: "23", profileId: "0104", inClusters: "0000,0001,0003,000F,0020,0500", outClusters: "000A,0019", deviceJoinName: "Aurora Smart PIR Sensor"
```

Comment:
We don't have many fingerprints with `endpoint`. Did you check to make sure this fingerprint is only applying to devices with an endpoint of 0x23? For example, you should be able to change the endpoint here to something else (ex: 22) and then verify the device joins as a Thing and not as a `Motion Detector`.

Reply:
Actually now that I read the ticket can't you add a normal fingerprint with manufacturer: Aurora model: MotionSensor51AU?

Reply:
Yeah. @KKlimczukS this is the fingerprint I use to get the device to add with the actual mfr/model in the IDE. Once you know that you can just make it an mfr/model fingerprint as usual.

Reply:
Turned out that device is paired with an endpoint 22 when i used a new fingerprint: 
fingerprint mfr: "Aurora", model: "MotionSensor51AU", deviceJoinName: "Aurora Smart PIR Sensor" (so it doesn't matter if I put there any other endpoint number).
It's possible to pair with specific endpoint only when You use old way fingerprint (put specific clusters in fingerprint), but writing an endpoint number in fingerprint doesn't matter either.